src/modules/ens/sagas.spec.ts

```diff
@@ -246,3 +249,46 @@ describe('when handling the fetching of external ens names for an owner', () =>
     })
   })
 })
+
+describe('when handling the fetch ens world status request', () => {
+  let subdomain: string
+
+  describe('when the subdomain provided is a dcl subdomain', () => {
+    beforeEach(() => {
+      subdomain = 'name.dcl.eth'
+    })
+
+    it('should get the ens object by using the getENSBySubdomain selector', async () => {
+      await expectSaga(ensSaga, builderClient, ensApi)
+        .select(getENSBySubdomain, subdomain)
+        .dispatch(fetchENSWorldStatusRequest(subdomain))
```

Comment:
The main objective of the sagas tests is to check the correctness of the actions that are put (which is the objective of any saga), would you mind checking what is put?